pkg/systemd/quadlet/quadlet.go

```diff
@@ -287,13 +287,11 @@ func ConvertContainer(container *parser.UnitFile, isUser bool) (*parser.UnitFile
 	// Need the containers filesystem mounted to start podman
 	service.Add(UnitGroup, "RequiresMountsFor", "%t/containers")
 
-	// If the conman exited uncleanly it may not have removed the container, so force it,
-	// -i makes it ignore non-existing files.
+	// If conmon exited uncleanly it may not have removed the container, so
+	// force it, -i makes it ignore non-existing files.
+	service.Add(ServiceGroup, "ExecStop", podmanBinary()+" rm -f -i --cidfile=%t/%N.cid")
 	service.Add(ServiceGroup, "ExecStopPost", "-"+podmanBinary()+" rm -f -i --cidfile=%t/%N.cid")
```

Comment:
Why keep the `ExecStopPost`? This is the exact same command so it should be removed.

Reply:
If the main PID gets killed, `ExecStop` will not be executed but the `*Post`.

Reply:
Done :heavy_check_mark: